Several pages still use removed services

mburke5678 opened the pull request:
https://bugzilla.redhat.com/show_bug.cgi?id=1614095

Also: https://bugzilla.redhat.com/show_bug.cgi?id=1615199

nekop commented on install_config/configuring_sdn.adoc:
```diff
@@ -138,9 +138,12 @@ networkConfig:
+is using that CIDR range, but be sure to restart the API and master services for any changes to take effect.
+
+----
+master-restart api master-restart controllers
```
Should be separated 2 lines.

```
master-restart api
master-restart controllers
```

nekop commented on install_config/configuring_sdn.adoc:
```diff
@@ -271,24 +274,53 @@ If you are already using one SDN plug-in and want to switch to another:
+. Restart the API and master services on all masters.
++
+----
+master-restart api master-restart controllers
```
Should be separated 2 lines.

```
master-restart api
master-restart controllers
```

nekop commented on install_config/configuring_sdn.adoc:
```diff
@@ -271,24 +274,53 @@ If you are already using one SDN plug-in and want to switch to another:
+----
 endif::[]
+
 . If you are switching between OpenShift SDN plug-ins, restart the *openvswitch* service on all masters and nodes.
```
It's no longer "openvswitch service". We can simply say "restart OpenShift SDN."

mburke5678 commented:
@nekop Can you please take another look? I have made further changes, such as the API health check, replaced stopping and disabling service.,

nekop commented on admin_guide/ipsec.adoc:
```diff
@@ -53,21 +53,25 @@ required.
-# ovs-vsctl del-br br0
-# systemctl start atomic-openshift-node
+# mkdir -p /etc/origin/node/pods-stopped
+# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
```
- The static pod files are master services, not the node we want to reboot
- If we move files, we need to restore the files after that
- We don't need to reboot node services nor del-br br0 if we reboot the node at step 3

To fix, simply replace the steps with node reboot.

nekop commented on admin_guide/iptables.adoc:
```diff
@@ -103,7 +103,7 @@ To make permanent rules changes, edit the changes into this file. Do not include
 
 After *iptables.service* is started or restarted on a node, the Docker service
-and *atomic-openshift-node.service* must be restarted to reconstruct the needed
+and node must be restarted to reconstruct the needed
```
The atomic-openshift-node.service is still used in 3.10, no need to replace it.

nekop commented on admin_guide/manage_nodes.adoc:
```diff
@@ -750,10 +750,17 @@ On the node where you want to restart Docker storage:
 ----
 
-. Run the following command to shut down Docker and the *atomic-openshift-node* service:
+. Run the following command to shut down Docker:
```
The atomic-openshift-node.service is still used in 3.10, no need to replace it.

nekop commented on admin_guide/manage_nodes.adoc:
```diff
@@ -750,10 +750,17 @@ On the node where you want to restart Docker storage:
++
+----
+# mkdir -p /etc/origin/node/pods-stopped
+# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
```
Revert.

nekop commented on admin_guide/manage_nodes.adoc:
```diff
@@ -789,10 +796,16 @@ $ docker-storage-setup
+$ systemctl start docker
+----
+
+. Restart the node service by rebooting the host:
```
Revert.

nekop commented on admin_guide/manage_nodes.adoc:
```diff
@@ -789,10 +796,16 @@ $ docker-storage-setup
 +
 ----
-$ systemctl start docker atomic-openshift-node
+# reboot
```
Revert.

nekop commented on admin_guide/out_of_resource_handling.adoc:
```diff
@@ -135,20 +135,6 @@ kubeletArguments:
-----
-endif::[]
-
-
```
Revert.

nekop commented on admin_guide/out_of_resource_handling.adoc:
```diff
@@ -375,21 +361,6 @@ kubeletArguments:
-endif::[]
-
-
-
```
Revert.

nekop commented on admin_guide/out_of_resource_handling.adoc:
```diff
@@ -415,20 +386,6 @@ Use the default value (5 minutes) before doing any adjustments.
-----
-endif::[]
-
-
```
Revert.

nekop commented on admin_guide/seccomp.adoc:
```diff
@@ -78,19 +78,6 @@ kubeletArguments:
-# systemctl restart origin-node
-----
-endif::[]
-
```
Revert.

nekop commented on admin_guide/seccomp.adoc:
```diff
@@ -124,19 +111,6 @@ kubeletArguments:
-# systemctl restart origin-node
-----
-endif::[]
-
```
Revert.

nekop commented on admin_guide/sysctls.adoc:
```diff
@@ -121,19 +121,6 @@ kubeletArguments:
-# systemctl restart origin-node
-----
-endif::[]
-
```
Revert.

nekop commented on admin_guide/topics/proc_etcd-quorum-single-node-v2-v3.adoc:
```diff
@@ -15,10 +15,13 @@ remaining etcd node a standalone etcd cluster.
-# systemctl stop etcd.service
+# mkdir -p /etc/origin/node/pods-stopped
+# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
+# reboot
```
We need to restore files before reboot and to restore files we need to stop atomic-openshift-node.service (otherwise it starts again).

```
# mkdir -p /etc/origin/node/pods-stopped
# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
# systemctl stop atomic-openshift-node
# mv /etc/origin/node/pods-stopped/* /etc/origin/node/pods/
# reboot
```

We probably need more steps to see if etcd is properly stopped. I'll send a query to openshift-sme.

lihongan replied:
Seems we just need stop etcd service here, so I think should move etcd.yaml instead of all files in `/etc/origin/node/pods/` folder unless you want stop master-api and controllers too. 
so it might be:
`mv /etc/origin/node/pods/etcd.yaml /etc/origin/node/pods-stopped/`
@xingxingxia @geliu2016 could you help confirm?

geliu2016 replied:
@lihongan yes, AGREE.

lihongan commented on day_two_guide/topics/api_service_status.adoc:
```diff
@@ -6,28 +6,22 @@ Module included in the following assemblies:
-master instances. To see the status of the service:
+The OpenShift API service runs on all
+master instances. To see the status of the service,
+view the pods in the *kube-system* project:
```
There are three types of pods: api, controllers, and etcd in the 'kube-system' project:
If here focus API service, then you can say `view the master-api pods in the *kube-system* project` and use the command with a label to get master-api pod only as below:
` oc get pod -n kube-system -l openshift.io/component=api`

lihongan commented on day_two_guide/topics/api_service_status.adoc:
```diff
@@ -6,28 +6,22 @@ Module included in the following assemblies:
+NAME                             READY     STATUS    RESTARTS   AGE
+master-api.example.com           1/1       Running   0          56d
+master-controllers.example.com   1/1       Running   0          56d
+master-etcd.example.com          1/1       Running   0          56d
```
The master-controllers and master-etcd pod in the output can be removed if using the command above with the label.

lihongan commented on day_two_guide/topics/api_service_status.adoc:
```diff
@@ -6,28 +6,22 @@ Module included in the following assemblies:
 ----
-$ curl -k https://master.example.com/healthz
+$ curl -k https://master-api.example.com/healthz
 ok
```
It might not be true. The `master-api.example.com` looks like the pod name and you cannot curl it. You should get the hostname or IP of the node which the pod landed then curl the host. for example:
```
$ oc get pod -n kube-system -o wide
NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
master-api-qe-anli310master-etcd-zone1-1           1/1       Running   0          7h        10.240.0.16   qe-anli310master-etcd-zone1-1
$ curl -k https://qe-anli310master-etcd-zone1-1/healthz
ok
```

mburke5678 commented:
@nekop @lihongan Thank you very much for your comments!! Very helpful.Can you please review the changes and make sure I addressed all of your concerns?

mburke5678 commented:
@xingxingxia Thank you for working late to review this PR and apologies for the difficulty of this crossing different groups. I greatly appreciate your help with this. 

cc @wsun1 @xltian

mburke5678 commented on admin_guide/topics/proc_etcd-quorum-single-node-v2-v3.adoc:
```diff
@@ -15,10 +15,15 @@ remaining etcd node a standalone etcd cluster.
+# mv /etc/origin/node/pods/etcd.yaml /etc/origin/node/pods-stopped/
+# systemctl stop atomic-openshift-node
+# mv /etc/origin/node/pods-stopped/etcd.yaml /etc/origin/node/pods/
+# reboot
```
@xingxingxia Thank you. This PR is for 3.10=>

geliu2016 replied:
hello @mburke5678, regarding to 'reboot', I tried to reboot etcd host and 'systemctl start atomic-openshift-node', both works(the etcd pod status is running), so my question is reboot etcd host is mandatory or run 'systemctl start atomic-openshift-node' is also ok.

as my experience, sometimes, in some special vm(virtual machine) env, reboot host operation will result in the host can't be started normally, even be terminated. so i have some concern about reboot operation.

mburke5678 replied:
@geliu2016 I removed the reboot command so that the user need to only restate the node. I will ask others to be certain.

geliu2016 replied:
@mburke5678 , for etcd issue, you may try to ask to be cretain with jlegasse@redhat.com

mburke5678 replied:
@geliu2016 Joe Legasse: "As far as etcd is concerned, no reboot is necessary of the host machine..."

lihongan commented on day_two_guide/topics/api_service_status.adoc:
```diff
@@ -6,28 +6,24 @@ Module included in the following assemblies:
-$ curl -k https://master.example.com/healthz
+oc get pod -n kube-system -o wide
+NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
+master-api-myserver.com                            1/1       Running   0          7h        10.240.0.16   myserver.com/healthz
```
The node name is `myserver.com` so please remove `/healthz` from it.

lihongan replied:
Other fixes here looks good, thanks.

mburke5678 commented:
@nekop @xingxingxia @lihongan @geliu2016 Are we OK to send this PR to documentation peer review?

mburke5678 commented:
@nekop @xingxingxia @lihongan @geliu2016 Are we OK to send this PR to documentation peer review?

mburke5678 commented:
@geliu2016 Did I address your comment? Was it https://github.com/openshift/openshift-docs/pull/11482#discussion_r214611020?

mburke5678 commented:
@mfojtik Can you help with an ectd question? In the "Creating a single-node etcd cluster" procedure, does the user need a reboot after stopping etcd in Step 1 of the following? 
 https://github.com/openshift/openshift-docs/blob/45165857d06f12c2d6d185a6078804ee9656aa50/admin_guide/topics/proc_etcd-quorum-single-node-v2-v3.adoc

mburke5678 commented:
@csheremeta PTAL

mburke5678 commented:
@geliu2016 @lihongan PTAL One change from a customer issue and we can merge.
https://github.com/openshift/openshift-docs/pull/11482/commits/ca1eb2f881c91bd24ab37a150d3eb9c2f48d5a6b

geliu2016 commented:
@mburke5678, LGTM, thx

mburke5678 commented:
@openshift/team-documentation Find and replace changes. PTAL to see if there is anything I did wrong.

mburke5678 commented:
/cherrypick enterprise-3.10

mburke5678 commented:
/cherrypick enterprise-3.11
